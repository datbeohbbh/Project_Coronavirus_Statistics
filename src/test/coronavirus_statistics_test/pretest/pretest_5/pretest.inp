8 15
q 1 1 20
q 1 2 10
q 2 2 9
? 4 2
? 4 1
q 1 5 8
q 3 1 1
q 2 5 2
q 2 2 1
q 1 6 41
? 4 2
? 7
? 4 1
? 4 2
? 4 3