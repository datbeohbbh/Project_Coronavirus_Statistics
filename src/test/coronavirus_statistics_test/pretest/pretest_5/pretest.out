Query ? 4 2 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 20
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 2 
- Information of the city 2 -
-> Number of current patients : 1
-> Has recovered : 9
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 19
-> Has recovered : 0
-> Was dead : 1

Query ? 7 :
The city with the highest death is city with the index 1
Quantity: 1 people

Query ? 4 1 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 6
-> Has recovered : 2
-> Was dead : 0

Query ? 4 2 :
The answer is the city with the index 1 
- Information of the city 1 -
-> Number of current patients : 19
-> Has recovered : 0
-> Was dead : 1

Query ? 4 3 :
The answer is the city with the index 6 
- Information of the city 6 -
-> Number of current patients : 41
-> Has recovered : 0
-> Was dead : 0

